101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -159,6 +159,10 @@
                 {
                   "name": "ContactEMailForLetsEncrypt",
                   "value": "[parameters('letsEncryptMail')]"
+                },
+                {
+                  "name": "artifacturl",
+                  "value": "https://bcartifacts.azureedge.net/onprem/11.0.46855.0/w1"
```

Comment:
this seems fragile - what is it for?

Reply:
ok, I suspect it will break again in the future if that's not a permanent artifact store... but let's leave it if it's working now.